internal/presentation/presentation.go

```diff
@@ -458,6 +475,18 @@ func prettyMetrics(w io.Writer, m metrics.Metrics, limit int) error {
 	return nil
 }
 
+var statKeys = []string{"min", "max", "mean", "stddev", "median", "75%", "90%", "95%", "99%", "99.9%", "99.99%"}
```

Comment:
Regarding the table..what if we do only `min`, `mean`, `max`, `90%`, `99%`, `99.9%` and maybe reduce the precision.

Reply:
That's a good idea, thanks. I suppose we could make it configurable, but if we just keep the JSON format with all the details, then that'll be what to reach for when the pretty abbreviated output isn't enough...